src/client/clientmap.cpp

```diff
@@ -218,17 +223,23 @@ void ClientMap::updateDrawList()
 				continue;
 			}
 
-			float range = 100000 * BS;
-			if (!m_control.range_all)
-				range = m_control.wanted_range * BS;
+			v3s16 block_coord = block->getPos();
+			v3s16 block_position = block_coord * MAP_BLOCKSIZE + MAP_BLOCKSIZE / 2;
 
-			float d = 0.0;
-			if (!isBlockInSight(block->getPos(), camera_position,
-					camera_direction, camera_fov, range, &d))
-				continue;
+			// First, perform a simple distance check.
+			if (!m_control.range_all &&
+					block_position.getDistanceFrom(cam_pos_nodes) > range + UNLOAD_PADDING)
+				continue; // Out of range, skip.
 
+			// This block is in range. Reset usage timer.
```

Comment:
Should mention that this is intentionally done *before* culling.

Reply:
This block is **_in range_**

Reply:
...but I cleared up the comments a bit anyway. Now that isBlockInSight is labeled as a frustum cull step, nobody in their right mind should attempt to tie block usage to that.